components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/constant/OrganizationManagementConstants.java

```diff
@@ -474,7 +474,17 @@ public enum ErrorMessages {
         ERROR_CODE_ERROR_FIRING_EVENTS("65090", "Error while firing the events",
                 "Server encountered an error while firing the events"),
         ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_DEPTH("65091", "Unable to retrieve the organization depth.",
-                "Server encountered an error while retrieving depth of organization with ID: %s.");
+                "Server encountered an error while retrieving depth of organization with ID: %s."),
+        ERROR_CODE_ERROR_RETRIEVING_ORGANIZATION_APPLICATIONS("65092", "Error while retrieving applications",
+                "Encountered an error while retrieving applications in organization id %s"),
+        ERROR_CODE_ERROR_CHECKING_APPLICATION_IS_A_FRAGMENT("65093", "Error while checking whether fragment app",
+                "Encountered an error while checking whether application %s is a fragment application"),
+        ERROR_CODE_INVALID_SHARE_APPLICATION_EMPTY_REQUEST_BODY("65094", "Invalid request",
+                "At least one of the attributes from shareWithAllChildren and sharedOrganizations should be present."),
+        ERROR_CODE_INVALID_SHARE_APPLICATION_REQUEST_BODY("65095", "Invalid request", "Cannot share application " +
+                "with a set of child organizations when shareWithAllChildren is set"),
+        ERROR_CODE_ERROR_UPDATING_APPLICATION_ATTRIBUTE("65096", "Error when updating the main application",
+        "Encountered an error when updating the main application of id: %s with property share with all children.");
```

Comment:
If this error is given due to the client error, move under client error enums and change the error code to `600..`

Reply:
This should belong to server errors.